jaffle_shop_online/models/marketing/schema.yml

```
@@ -143,6 +143,19 @@ models:
       - name: total_spend
         data_type: number
         description: "Total spend amount (USD)"
+        data_tests:
+          - elementary.column_anomalies:
+              anomaly_direction: both
+              anomaly_sensitivity: 3
+              column_anomalies:
+              - null_count
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 16 but found 14

(indentation)